Applications/DataExplorer/VtkVis/MeshFromRasterDialog.cpp

```diff
@@ -17,7 +17,7 @@
 #include "MeshGenerators/VtkMeshConverter.h"
 
 MeshFromRasterDialog::MeshFromRasterDialog(QDialog* parent)
-: QDialog(parent)
+: _mesh_name("mesh"), _array_name("Colour"), QDialog(parent)
```

Comment:
Looks ok, but there is a new warning (hover over the marked line):
https://svn.ufz.de:8443/user/endjunction/my-views/view/ogs6/job/OGS-6/job/Gui/job/Mac-PRs/1061/warnings7Result/source.-2648670318250268918/#10
```
field '_array_name' will be initialized after base 'QDialog'
```